programs/models.py

```diff
@@ -223,14 +234,22 @@ def from_model_data(self, data: DataType):
         program_category = self.instance
 
         program_category.calculator = data["calculator"]
-        program_category.icon = data["icon"]
+        if data["icon"]:
+            icon = CategoryIconName.objects.filter(name=data["icon"]).first()
+            if not icon:
+                icon = CategoryIconName.objects.create(name=data["icon"])
+            program_category.icon = icon
+        else:
+            program_category.icon = None
         program_category.priority = data["priority"]
```

Comment:
@CalebPena @msrezaie lines 237-243 are the lines I adjusted the logic for to get the translations to import.